src/Fixer/Whitespace/IndentationTypeFixer.php

```diff
@@ -33,7 +33,8 @@ public function getDefinition()
         return new FixerDefinition(
             'Code MUST use configured indentation type.',
             [new CodeSample("<?php\n\nif (true) {\n\techo 'Hello!';\n}"),
-        ]);
+        ]
+        );
```

Comment:
This is not the best fix, but the original code is very wrong in the first place...